src/object.rs

```diff
@@ -6,12 +6,12 @@ use crate::Context;
 use crate::Local;
 use crate::Name;
 use crate::Object;
-use crate::PropertyCallbackInfo;
+use crate::PropertyCallbackScope;
 use crate::ToLocal;
 use crate::Value;
 
 pub type AccessorNameGetterCallback =
-  extern "C" fn(Local<Name>, &PropertyCallbackInfo);
+  extern "C" fn(Local<Name>, PropertyCallbackScope);
```

Comment:
I should probably undo this experimental change before landing.
This is not the final form it'll take and also not heading in that direction.